app/components/collections/links_component.rb

```diff
@@ -3,15 +3,7 @@
 
 module Collections
   # Renders the header for the collection show page (title and create new link)
-  class LinksComponent < ApplicationComponent
-    sig { params(collection: Collection).void }
-    def initialize(collection:)
-      @collection = collection
-    end
-
-    sig { returns(Collection) }
-    attr_reader :collection
-
+  class LinksComponent < Collections::ShowComponent
```

Comment:
I don't understand why this is an inheritance.  This does not seem like natural inheritance to me.